package.json

```diff
@@ -66,6 +66,9 @@
       "perMachine": true
     }
   },
+  "scripts": {
+    "start": "./node_modules/.bin/electron src/main"
```

Comment:
does it work to just say "electron src/main" since electron is in the bin?

Reply:
It does, I like to show the full path so others know exactly whats going on

Reply:
I can change it to `electron src/main` if you think that looks better

Reply:
hmm no need, lets keep it as is then